models/migrations/v1_19/v238.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_19 //nolint
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+// AddCardTypeToProjectTable: add CardType column, setting existing rows to CardTypeTextOnly
+func AddCardTypeToProjectTable(x *xorm.Engine) error {
+	type Project struct {
+		CardType int `xorm:"NOT NULL DEFAULT 1"`
+	}
+
+	if err := x.Sync2(new(Project)); err != nil {
+		return fmt.Errorf("Sync2: %w", err)
+	}
```

Comment:
`Sync2` is deprecated.

```suggestion
	if err := x.Sync(new(Project)); err != nil {
		return fmt.Errorf("Sync: %w", err)
	}
```